### DISCUSSION TITLE
[Feature] Support OFFSET in Python interface

### DISCUSSION THREAD

**Discussion by wisp3rwind** [duckdb/duckdb#3711] [Ideas]
As the title says, it would be great if, in addition to `rel.limit(n)`, the relational python interface supported the `OFFSET` clause, for example as

```python
result = rel.limit(5, offset=100).execute()
```
or

```python
result = rel.limit(5).offset(100).execute()
```
As far as I can tell from https://github.com/duckdb/duckdb/blob/master/tools/pythonpkg/src/pyrelation.cpp, there's not `OFFSET` support right now.

Thanks!

**Comment by hannes** [comment]
Happy to review a PR, should not be too hard

**Reply by wisp3rwind** [reply]
> Happy to review a PR, should not be too hard

See https://github.com/duckdb/duckdb/pull/3739
